src/index.ts

```diff
@@ -14,15 +15,15 @@ import {RoutingControllersOptions} from "./RoutingControllersOptions";
 /**
  * Registers all loaded actions in your express application.
  */
-export function useExpressServer<T>(expressApp: T, options?: RoutingControllersOptions): T {
+export function useExpressServer(expressApp: express.Express, options?: RoutingControllersOptions): express.Express {
```

Comment:
According to [this section](https://github.com/pleerock/routing-controllers#using-exist-server-instead-of-creating-a-new-one), it seems useExpressServer takes a `expressExpress` variable because `let app = express();` it will get `express.Express` value. So, I think it's `express.Express`, not `express.Application`.